automation/default_browser_params.json

```diff
@@ -17,13 +16,8 @@
     "display_mode": "native",
     "browser": "firefox",
     "prefs": {},
-
     "tp_cookies": "always",
     "donottrack": false,
-    "disconnect": false,
-    "ghostery": false,
-    "https-everywhere": false,
     "adblock-plus": false,
```

Comment:
Just realizing, that we also have adblock-plus in here. Feel free to delete that as well.